src/index.html

```diff
@@ -192,10 +192,10 @@ <h2 class="modal__title" id="modal-card-delete-verify-title">
         </button>
       </form>
     </div>
- -->
+
     <!-- EDIT AVATAR PICTURE MODAL -->
 
```

Comment:
You can make ` setInputValues` in class `PopupWithForm`, which can insert data into inputs:

```js

  setInputValues(data) {
    this._inputList.forEach((input) => {
      // here you insert the `value` by the `name` of the input
      input.value = data[input.name];
    });
  }



```

So, this way you’ll not have to search the inputs of the profile in `index.js`